examples/nextjs-todo-list/pages/index.tsx

```diff
@@ -88,7 +102,7 @@ function Loading() {
 }
 
 export default function Page() {
-  const roomId = useOverrideRoomId("nextjs-todo-list");
+  const roomId = useOverrideRoomId("nextjs-todo-list-v2");
```

Comment:
Technically, the switch to `LiveObject`s here can possibly break when clicking on items that have previously been created as JSON todo items. The easiest fix for this was to just bump to a different room ID, but it's the first time we've done that in our examples, I think? Any advice? Worth it?

Reply:
We do [clean up unused-since-at-least-24h example rooms daily](https://github.com/liveblocks/liveblocks.io/blob/main/scripts/cron/clean-up-examples.ts), but that still leaves plenty of room for error so yeah I'd say it's worth it to do versioning.